closed/src/java.base/share/classes/com/sun/crypto/provider/NativeChaCha20Cipher.java

```diff
@@ -966,6 +995,10 @@ private final class EngineAEADEnc implements ChaChaEngine {
         public synchronized int doUpdate(byte[] in, int inOff, int inLen, byte[] out,
                 int outOff) throws ShortBufferException, KeyException {
             if (initialized) {
+                if (!opensslInitialized) {
+                    ChaCha20Init(false);
+                }
+
                 // If this is the first update since AAD updates, signal that
                 // we're done processing AAD info
```

Comment:
Please finish this sentence with a period.

Reply:
Done.